src/main/java/com/cb/services/dbService/dbServiceImpl/CharacterDBServiceImpl.java

```diff
@@ -25,10 +27,12 @@ public void setTemplate(JdbcTemplate template) {
             public CharacterDAL mapRow(ResultSet rs, int rowNum) throws SQLException {
                 CharacterDAL c = new CharacterDAL();
                 c.setName(rs.getString(2));
+                c.setImage(rs.getString(3));
```

Comment:
I would suggest to use mapper here instead of  writing all fields needed. We have to receive DB table mirror in the DAL.

Reply:
What do you mean exactly? Cause we all the time doing like this.

Reply:
The right approach:

    public List<UserDAL> getUsers() {
        return template.query("SELECT * FROM users", new BeanPropertyRowMapper(UserDAL.class));
    }

Reply:
Updated